include/envoy/network/connection_handler.h

```diff
@@ -72,7 +72,7 @@ class ConnectionHandler {
   /**
    * Used by ConnectionHandler to manage listeners.
    */
-  class ActiveListener {
+  class ActiveListener : public Listener {
```

Comment:
nit: I don't think we should derive from Listener if we expose a Listener member. Can we remove this part?

Reply:
removed, added disable/enable() to this interface instead.